src/suggestions/index.jsx

```diff
@@ -72,14 +74,16 @@ export const SuggestionsMixin = {
     window.addEventListener('keydown', this.handleSuggestionsKeyDown, false);
     window.addEventListener('keyup', this.handleSuggestionsKeyUp, false);
     window.addEventListener('blur', this.handleSuggestionBlur, true);
+
+    this.props.fetchSuggestions(this.props.note.meta.ids.site);
   },
 
   componentDidUpdate() {
-    if (!this.suggestionMixin_suggestionList) {
+    if (!this.suggestionsMixin_suggestionList) {
```

Comment:
We may just want to rename this later. Such a long and odd mix of casing here 😅

Reply:
yeah, I started moving this to a high order component because the `mixing` thing it's so `2016`, but the diff was going to be huge. When we move it to the `hoc`, this will be so much clearer.